runelite-client/src/main/java/net/runelite/client/plugins/wasdcamera/WASDCameraInputListener.java

```diff
@@ -52,24 +52,9 @@ public void keyPressed(KeyEvent e)
 	{
 		//log.debug("Pressed: " + e);
 
-		if (!plugin.isInGame())
-		{
-			return;
-		}
-
-		if (plugin.isChatMode())
-		{
-			switch (e.getKeyCode())
-			{
-				case KeyEvent.VK_ESCAPE:
-					e.consume();
-				case KeyEvent.VK_ENTER:
-					plugin.setChatMode(false);
-					break;
-			}
-		}
-		else
+		if (plugin.shouldRemap())
```

Comment:
Merge this commit with previous commit

Reply:
I will rebase the PR when everything works